packages/radix/src/GlobalStyles.tsx

```diff
@@ -41,11 +41,21 @@ export const GlobalStyles = createGlobalStyle(
         fontDisplay: 'swap',
         src: `local('UntitledSans'), url(${UntitledSansWoff2}) format('woff2'), url(${UntitledSansWoff}) format('woff')`,
       },
+      {
+        fontFamily: 'UntitledSans',
+        fontWeight: '500',
```

Comment:
The quotes around `500` is not needed since it's a number. To keep it consistent with the rest of the codebase, can we remove them please?

Reply:
Done https://github.com/modulz/radix/pull/130/commits/f59102d7d9f7a4bee7191c3801db8cb79065237f